xml/signals/FS-1987/appearance-segnalebasso.xml

```diff
@@ -41,6 +41,12 @@
        <authorinitials>PDB</authorinitials>
        <revremark>Second version for Italian Signal System</revremark>
      </revision>
+     <revision>
+       <revnumber>3</revnumber>
+       <date>2023-08-23</date>
+       <authorinitials>PDB</authorinitials>
+       <revremark>Third version for Italian Signal System - amended aspect mapping</revremark>
+     </revision>
```

Comment:
This change seems unnecessary since it's the only change in this file. But no need to change.